CCP_App/src/test/java/com/example/demo2/service/UserServiceTest.java

```diff
@@ -0,0 +1,104 @@
+package com.example.demo2.service;
+
+import com.example.demo2.custom.exception.NonUpdateFieldException;
+import com.example.demo2.exception.handler.GlobalExceptionHandler;
+import com.example.demo2.model.User;
+import com.example.demo2.repository.UserRepository;
+import com.example.demo2.viewmodel.UserViewModel;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.*;
+
+/************************************************************************
+ *                                                                      *
+ *  DDDD     SSSS    AAA        Daten- und Systemtechnik Aachen GmbH    *
+ *  D   D   SS      A   A       Pascalstrasse 28                        *
+ *  D   D    SSS    AAAAA       52076 Aachen-Oberforstbach, Germany     *
+ *  D   D      SS   A   A       Telefon: +49 (0)2408 / 9492-0           *
+ *  DDDD    SSSS    A   A       Telefax: +49 (0)2408 / 9492-92          *
+ *                                                                      *
+ *                                                                      *
+ *  (c) Copyright by DSA - all rights reserved                          *
+ *                                                                      *
+ ************************************************************************
+ *
+ * Author : Riyaardh Adam
+ * Created: 2023/02/15 09:01
+ *
+ ***********************************************************************/
+
+public class UserServiceTest {
+
+	@Mock
+	UserRepository userRepository;
+
+	UserService service;
+	GlobalExceptionHandler exceptionHandler;
+
+	@BeforeEach
+	public void init() {
+		exceptionHandler = new GlobalExceptionHandler();
+		MockitoAnnotations.openMocks(this);
+		User user = new User("Riyaardh", "Adam", "DZAC6AG", "riyaardhadam@gmail.com");
+		List<User> mockData = new ArrayList<>();
+		mockData.add(user);
+		when(userRepository.findAll()).thenReturn((List<User>) mockData);
+		when(userRepository.findById(any(String.class))).thenReturn(Optional.of((User) user));
```

Comment:
try avoid the use of any() when you have better options available to you.
using a value or a matcher like eq(value) is a better practice to ensure your code works as intended and doesn't let errors slip through